src/Ocelot/Configuration/File/FileRoute.cs

```diff
@@ -55,77 +55,77 @@ public FileRoute(FileRoute from)
+        public FileLoadBalancerOptions LoadBalancerOptions { get; set; }
         public IDictionary<string, string> Metadata { get; set; }
-        public int Priority { get; set; }
-        public FileQoSOptions QoSOptions { get; set; }
-        public FileRateLimitRule RateLimitOptions { get; set; }
-        public string RequestIdKey { get; set; }
-        public Dictionary<string, string> RouteClaimsRequirement { get; set; }
-        public bool RouteIsCaseSensitive { get; set; }
-        public FileSecurityOptions SecurityOptions { get; set; }
-        public string ServiceName { get; set; }
-        public string ServiceNamespace { get; set; }
-        public int Timeout { get; set; }
-        public Dictionary<string, string> UpstreamHeaderTransform { get; set; }
-        public string UpstreamHost { get; set; }
-        public List<string> UpstreamHttpMethod { get; set; }
-        public string UpstreamPathTemplate { get; set; }
-        public IDictionary<string, string> UpstreamHeaderTemplates { get; set; }
-
-        /// <summary>
-        /// Clones this object by making a deep copy.
-        /// </summary>
-        /// <returns>A <see cref="FileRoute"/> deeply copied object.</returns>
-        public object Clone()
-        {
-            var other = (FileRoute)MemberwiseClone();
-            DeepCopy(this, other);
-            return other;
-        }
-
-        public static void DeepCopy(FileRoute from, FileRoute to)
-        {
-            to.AddClaimsToRequest = new(from.AddClaimsToRequest);
-            to.AddHeadersToRequest = new(from.AddHeadersToRequest);
-            to.AddQueriesToRequest = new(from.AddQueriesToRequest);
-            to.AuthenticationOptions = new(from.AuthenticationOptions);
-            to.ChangeDownstreamPathTemplate = new(from.ChangeDownstreamPathTemplate);
-            to.DangerousAcceptAnyServerCertificateValidator = from.DangerousAcceptAnyServerCertificateValidator;
-            to.DelegatingHandlers = new(from.DelegatingHandlers);
-            to.DownstreamHeaderTransform = new(from.DownstreamHeaderTransform);
-            to.DownstreamHostAndPorts = from.DownstreamHostAndPorts.Select(x => new FileHostAndPort(x)).ToList();
-            to.DownstreamHttpMethod = from.DownstreamHttpMethod;
-            to.DownstreamHttpVersion = from.DownstreamHttpVersion;
+        public int Priority { get; set; }
+        public FileQoSOptions QoSOptions { get; set; }
+        public FileRateLimitRule RateLimitOptions { get; set; }
+        public string RequestIdKey { get; set; }
+        public Dictionary<string, string> RouteClaimsRequirement { get; set; }
+        public bool RouteIsCaseSensitive { get; set; }
+        public FileSecurityOptions SecurityOptions { get; set; }
+        public string ServiceName { get; set; }
+        public string ServiceNamespace { get; set; }
+        public int? Timeout { get; set; }
+        public Dictionary<string, string> UpstreamHeaderTransform { get; set; }

```

Comment:
```suggestion
        public IDictionary<string, string> UpstreamHeaderTransform { get; set; }

```

Reply:
Not an issue.
It is beyond the scope of Timeout changes!